manifests/m/Microsoft/VisualStudio/2022/Professional/17.1.5/Microsoft.VisualStudio.2022.Professional.yaml

```diff
@@ -0,0 +1,30 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
+PackageIdentifier: Microsoft.VisualStudio.2022.Professional
+PackageVersion: 17.1.5
+PackageName: Visual Studio Professional 2022
+Publisher: Microsoft Corporation
+PackageUrl: https://visualstudio.microsoft.com/
+License: Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate